### PR TITLE
allowing to specify a custom work group for AWS Athena queries

### PR DESCRIPTION
**Pull request by ialeinikov**
## What type of PR is this? (check all applicable)

- [X] Feature

## Description
This change allows specifying a custom AWS Athena work group.

## Related Tickets & Documents

https://docs.aws.amazon.com/athena/latest/ug/workgroups.html

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
Screen Shot 2019-03-15 at 1 20 57 PM

### CODE REVIEW

File: redash/query_runner/athena.py [hunk 1]
```diff
@@ -78,6 +78,11 @@ def configuration_schema(cls):
                     'type': 'boolean',
                     'title': 'Use Glue Data Catalog',
                 },
+                'work_group': {
```
Comment (rauchy) [inline redash/query_runner/athena.py]: Could you add this to the correct place in `order`? (it happens now automatically due to the alphabetical order, but who knows what will come next)

File: redash/query_runner/athena.py [hunk 2]
```diff
@@ -78,6 +78,11 @@ def configuration_schema(cls):
...
+                    'type': 'string',
+                    'title': 'Athena work group',
+                    'default': 'primary'
```
Comment (arikfr) [inline redash/query_runner/athena.py]: Wouldn't it be safer/more backward compatible to just not pass anything if no value provided?

File: redash/query_runner/athena.py [hunk 3]
```diff
@@ -78,6 +78,11 @@ def configuration_schema(cls):
                     'type': 'boolean',
                     'title': 'Use Glue Data Catalog',
                 },
+                'work_group': {
+                    'type': 'string',
+                    'title': 'Athena work group',
```
Comment (arikfr) [inline redash/query_runner/athena.py]:
```suggestion
                    'title': 'Athena Work Group',
```

File: redash/query_runner/athena.py (hunk 2 above)
Comment (ialeinikov) [reply redash/query_runner/athena.py]: According to the AWS [documentation](https://docs.aws.amazon.com/athena/latest/ug/user-created-workgroups.html):

> By default, each account has a primary workgroup and the default permissions allow all authenticated users access to this workgroup. The primary workgroup cannot be deleted.

**Comment by ialeinikov** [comment]
@rauchy, this was tested on our STG instance of Redash with Athena connections.
I've tested it with providing custom values to the "work group" field (based on our configuration) and without any custom values in that field.

**Comment by rauchy** [comment]
@ialeinikov great! Have you given PyAthena a sanity check to make sure nothing breaks? (I'm referring to [this change](https://github.com/getredash/redash/pull/3592/files#diff-aab4aeb938d015cca5784d2c3bcea65dR23))

**Comment by ialeinikov** [comment]
@rauchy, yes, I ran queries internally. Using both specific work group datasource and the one with the default value. Haven't noticed any issues.

**Comment by arikfr** [comment]
Thanks!

**Pull request merged**